### ISSUE TITLE
{% include %} tags do not accept arbitrary expressions

### ISSUE THREAD

**Issue by wardourdigital** [ogonkov/nunjucks-loader#115]
**Environment**
Node version: v14.16.0
simple-nunjucks-loader version: 3.1.0
webpack version: 5.37.1

Webpack

```
			{
				test: /\.njk|html$/,
				use: [
					{
						loader: "simple-nunjucks-loader",
						options: {
							searchPaths: ["src-local-nunjucks/templates", "src-hubspot-theme/modules"],
							filters: {
								filter: path.join(__dirname, "filter.js"),
							},
						},
					},
				],
			},
```

**Describe the bug**
According to the documentation of nunjucks:

_include actually accepts any arbitrary expression, so you can pass anything into it, as long as the expression evaluates to a string or a compiled Template object: {% include name + ".html" %}._
[Include](https://mozilla.github.io/nunjucks/templating.html#include)

When I attempted to do this:

```
{% macro getBlock(name) %}
	{% include name + '-module/module.html' %}
{% endmacro %}

{{ getBlock("card-section") }}
```

I get the error:

```
ERROR in Error: Child compilation failed:
  Module build failed (from ./node_modules/simple-nunjucks-loader/lib/cjs/loader  .js):
  TypeError: Cannot read property 'split' of undefined
```

**Expected behavior**
For the named template to be included into the page.

**Comment by ogonkov** [comment]
Thank you for your report, will look at this issue on the weekend.

**Comment by ogonkov** [comment]
I have add warnings for such imports. Full support is on the way.

**Comment by jacobfentress** [comment]
This would be a huge help to have supported. Any news or tips for workarounds?

**Comment by ogonkov** [comment]
I have started to refactor imports/assets in #121 to use same flow for both to load dynamic assets, but it's still WIP.

**Comment by jakefentress** [comment]
A little bump here. Is this still under development?

**Comment by ogonkov** [comment]
Yes, it still priority. Loader code is a bit of mess. I planned to convert it to TS first, to have more control over all the abstractions.

**Comment by ArmandoAmador** [comment]
@wardourdigital @jacobfentress

Did anyone figure out a solution for this?

I'm also having the same issue when including a named template inside a Macro.

```
ModuleBuildError: Module build failed (from ./node_modules/simple-nunjucks-loader/lib/cjs/loader.js):
TypeError: Cannot read property 'split' of undefined
```

**Comment by ogonkov** [comment]
Hello, still on it. Sorry for delay.
